runelite-client/src/main/java/net/runelite/client/plugins/worldmap/QuestStartLocation.java

```diff
@@ -27,151 +27,162 @@
+// quests start in the same location, an array of quests is expected.
 enum QuestStartLocation
 {
 	//Free Quests
-	COOKS_ASSISTANT_RFD("Cook's Assistant", new WorldPoint(3211, 3216, 0)),
-	THE_CORSAIR_CURSE("The Corsair Curse", new WorldPoint(3029, 3273, 0)),
-	DEMON_SLAYER("Demon Slayer", new WorldPoint(3204, 3424, 0)),
-	DORICS_QUEST("Doric's Quest", new WorldPoint(2952, 3450, 0)),
-	DRAGON_SLAYER("Dragon Slayer", new WorldPoint(3190, 3362, 0)),
-	ERNEST_THE_CHICKEN("Ernest the Chicken", new WorldPoint(3109, 3330, 0)),
-	GOBLIN_DIPLOMACY("Goblin Diplomacy", new WorldPoint(2957, 3509, 0)),
-	IMP_CATCHER("Imp Catcher", new WorldPoint(3108, 3160, 0)),
-	THE_KNIGHTS_SWORD("The Knight's Sword", new WorldPoint(2976, 3342, 0)),
-	MISTHALIN_MYSTERY("Misthalin Mystery", new WorldPoint(3234, 3155, 0)),
-	PIRATES_TREASURE("Pirate's Treasure", new WorldPoint(3051, 3252, 0)),
-	PRINCE_ALI_RESCUE("Prince Ali Rescue", new WorldPoint(3301, 3163, 0)),
-	THE_RESTLESS_GHOST("The Restless Ghost", new WorldPoint(3240, 3210, 0)),
-	RUNE_MYSTERIES("Rune Mysteries", new WorldPoint(3210, 3220, 0)),
-	SHEEP_SHEARER("Sheep Shearer", new WorldPoint(3190, 3272, 0)),
-	SHIELD_OF_ARRAV_PHOENIX_GANG("Shield of Arrav (Phoenix Gang)", new WorldPoint(3208, 3495, 0)),
-	SHIELD_OF_ARRAV_BLACK_ARM_GANG("Shield of Arrav (Black Arm Gang)", new WorldPoint(3208, 3392, 0)),
-	VAMPIRE_SLAYER("Vampire Slayer", new WorldPoint(3096, 3266, 0)),
-	WITCHS_POTION("Witch's Potion", new WorldPoint(2967, 3203, 0)),
-	X_MARKS_THE_SPOT("X Marks the Spot", new WorldPoint(3227, 3242, 0)),
+	COOKS_ASSISTANT_RFD(Quest.COOKS_ASSISTANT, new WorldPoint(3211, 3216, 0)),
+	THE_CORSAIR_CURSE(Quest.THE_CORSAIR_CURSE, new WorldPoint(3029, 3273, 0)),
+	DEMON_SLAYER(Quest.DEMON_SLAYER, new WorldPoint(3204, 3424, 0)),
+	DORICS_QUEST(Quest.DORICS_QUEST, new WorldPoint(2952, 3450, 0)),
+	DRAGON_SLAYER(Quest.DRAGON_SLAYER, new WorldPoint(3190, 3362, 0)),
+	ERNEST_THE_CHICKEN(Quest.ERNEST_THE_CHICKEN, new WorldPoint(3109, 3330, 0)),
+	GOBLIN_DIPLOMACY(Quest.GOBLIN_DIPLOMACY, new WorldPoint(2957, 3509, 0)),
+	IMP_CATCHER(Quest.IMP_CATCHER, new WorldPoint(3108, 3160, 0)),
+	THE_KNIGHTS_SWORD(Quest.THE_KNIGHTS_SWORD, new WorldPoint(2976, 3342, 0)),
+	MISTHALIN_MYSTERY(Quest.MISTHALIN_MYSTERY, new WorldPoint(3234, 3155, 0)),
+	PIRATES_TREASURE(Quest.PIRATES_TREASURE, new WorldPoint(3051, 3252, 0)),
+	PRINCE_ALI_RESCUE(Quest.PRINCE_ALI_RESCUE, new WorldPoint(3301, 3163, 0)),
+	THE_RESTLESS_GHOST(Quest.THE_RESTLESS_GHOST, new WorldPoint(3240, 3210, 0)),
+	RUNE_MYSTERIES(Quest.RUNE_MYSTERIES, new WorldPoint(3210, 3220, 0)),
+	SHEEP_SHEARER(Quest.SHEEP_SHEARER, new WorldPoint(3190, 3272, 0)),
+	SHIELD_OF_ARRAV(Quest.SHIELD_OF_ARRAV, new WorldPoint(3208, 3495, 0)),
+	VAMPIRE_SLAYER(Quest.VAMPIRE_SLAYER, new WorldPoint(3096, 3266, 0)),
+	WITCHS_POTION(Quest.WITCHS_POTION, new WorldPoint(2967, 3203, 0)),
+	// X_MARKS_THE_SPOT("X Marks the Spot", new WorldPoint(3227, 3242, 0)),
```

Comment:
What is this? Missing from `Quests`?

Reply:
Ah, yes I believe it is. I'll add the enum you provided. Where do you reference the quest id from tho?